src/popup/App.vue

```diff
@@ -121,6 +132,44 @@ export default defineComponent({
       || routeMeta.value?.hideHeader
     ));
 
+    const NOTIFICATION_DEFAULT_SETTINGS: INotificationSetting[] = [
```

Comment:
Why this const was moved?

Reply:
I see the reason why you moved this - probably you didn't like the use of function `tg` in the constants. I agree this is bad. But I think we will move it when we will be working on moving the notification settings from vuex to composable. So for now I suggest to leave it as it is.

Reply:
I moved it because I was rewriting and moving the i18n component so I felt that it was reasonable to move it